app/models/organization.rb

```diff
@@ -132,12 +136,30 @@ def scale_values
     }
   end
 
-  def self.seed_items(org)
-    Rails.logger.info "Seeding #{org.name}'s items..."
-    org_id = org.id
-    base_items = BaseItem.pluck(:partner_key, :name).collect { |c| { partner_key: c[0], name: c[1], organization_id: org_id } }
-    Item.create(base_items)
-    org.reload
+  def self.seed_items(organization = Organization.all)
+    base_items = BaseItem.all.map(&:to_h)
+    Array.wrap(organization).each do |org|
+      Rails.logger.info "\n\nSeeding #{org.name}'s items...\n"
+      org.seed_items(base_items)
+      org.reload
+    end
+  end
+
+  def seed_items(item_collection)
+    Array.wrap(item_collection).each do |item|
+      items.create!(item)
+    rescue ActiveRecord::RecordInvalid => invalid
+      Rails.logger.info "[SEED] Duplicate item! #{invalid.record.name}"
+      existing_item = items.find_by(name: invalid.record.name)
+      if invalid.to_s.match(/been taken/).present? && existing_item.other?
+        Rails.logger.info "Changing Item##{existing_item.id} from Other to #{invalid.record.partner_key}"
+        existing_item.update(partner_key: invalid.record.partner_key)
+        existing_item.reload
+      else
```

Comment:
This block handles when an organization has an `Item` already created (with "Other" as parent type) that is now a base item itself. The match will be flaky because it requires an exact match, but it's better than nothing. 

Future improvement might be creating a `migrate_item_from_other` that would do this same thing but that can be used on an individual item basis from the site_admin dashboard